assets/css/fonts.css

```diff
@@ -40,6 +40,20 @@
     src: url('/fonts/ExpensifyMono-Bold.woff2') format('woff2'),  url('/fonts/ExpensifyMono-Bold.woff') format('woff');
 }
 
+@font-face {
+    font-family: ExpensifyNewKansas-Medium;
+    font-weight: 500;
+    font-style: normal;
+    src: url('/fonts/ExpensifyNewKansas-Medium.woff2') format('woff2'),  url('/fonts/ExpensifyNewKansas-Medium.woff') format('woff');
+}
+
+@font-face {
+    font-family: ExpensifyNewKansas-MediumItalic;
+    font-weight: 500;
+    font-style: italic;
+    src: url('/fonts/ExpensifyNewKansas-MediumItalic.woff2') format('woff2'),  url('/fonts/ExpensifyNewKansas-MediumItalic.woff') format('woff');
+}
```

Comment:
I think we can actually use the same value for font-family here. 
Then when we want to use the italic version we can put `font-style: italic` in the style object that's being applied to the element. LMK if that makes sense, this is something I just figured out today on [this PR](https://github.com/Expensify/App/pull/12457/files). You can see an example there with bold fonts. 

In general, I think this means we should refactor the fonts to follow this technique. If you don't want to do it here, we can go with your current code, and then I'll open a PR later to clean up all the fonts at once.

Reply:
Good to know 😄 I can update `New Kansas` to use the same font-family since we are introducing it here, but I think it's best to clean up the other fonts in a separate PR to separate the concerns and chances for regressions.

Reply:
I agree. I'll make that follow up PR once this one is merge. Thanks @luacmartins

Reply:
Thank you!